superset-frontend/src/explore/components/controls/AdhocFilterControl.jsx

```diff
@@ -39,6 +38,7 @@ import {
 } from '../OptionControls';
 import Icon from '../../../components/Icon';
 import AdhocFilterPopoverTrigger from '../AdhocFilterPopoverTrigger';
+import DndWithHTML5Backend from '../../DndContextProvider';
```

Comment:
```suggestion
import DndWithHTML5Backend from 'src/explore/DndContextProvider';
```

Nit: replace this (and maybe other imports in this file, too) with absolute path.